Makefile

```diff
@@ -119,11 +118,8 @@ $(CHECK_FORMAT): force-non-root
 
 install: force-root
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 0755 -T $(CILIUM_ENVOY_BIN) $(DESTDIR)$(BINDIR)/cilium-envoy
-# Strip only non-debug builds
-ifeq "$(findstring -dbg,$(realpath bazel-bin))" ""
-	$(STRIP) $(DESTDIR)$(BINDIR)/cilium-envoy
-endif
+	$(INSTALL) -m 0755 -T $(CILIUM_ENVOY_BIN) $(DESTDIR)$(BINDIR)/cilium-envoy-unstripped
+	$(STRIP) -o $(DESTDIR)$(BINDIR)/cilium-envoy $(DESTDIR)$(BINDIR)/cilium-envoy-unstripped
```

Comment:
Won't this strip the wrong binary (`cilium-envoy-unstripped`, but `cilium-envoy should be stripped)?